README.md

```diff
@@ -63,6 +63,12 @@ The full test suite can be found at [https://github.com/openshift/openshift-test
 
 - optionally run `ORIGIN_TESTS_REPOSITORY=test.repository.com:5000/origin-tests make origin-tests-disconnected-environment` to mirror all the required test images to a container image repository and source required test images from your repository when running origin-tests in a disconnected environment.
 
+### Custom RPMs
+The custom RPMs utility allows to install RPMs from an external source on a node.
+RPMS_SRC (RPMs download source URL) must be provided.
+REMOVE_PACKAGES should be set when installing RT kernel RPMs to override the installed kernel packages.
+RPMS_NODE_ROLE is optional and defaults to 'node-role.kubernetes.io/worker'.
```

Comment:
missing here somewhere the instruction to set these and run `make custom-rpms` or something